UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.IntegrationTests/RadioNavigationalWarning/BaseRNWTest.cs

```diff
@@ -0,0 +1,172 @@
+﻿using FakeItEasy;
```

Comment:
Please correct file path/namespace/class name; all should be `RadioNavigationalWarnings` rather than `RadioNavigationalWarning`.

/UKHO.MaritimeSafetyInformation.IntegrationTests/`RadioNavigationalWarnings`/BaseRNWTest.cs
`RadioNavigationalWarningsTest`